BUILD

```diff
@@ -61,12 +61,12 @@ container_bundle(
         "gcr.io/{PROJECT_ID}/java-debian10/jetty:java11": "//java/jetty:jetty_java11_debian10",
         "gcr.io/{PROJECT_ID}/java-debian10/jetty:debug": "//java/jetty:jetty_java11_debug_debian10",
         "gcr.io/{PROJECT_ID}/java-debian10/jetty:java11-debug": "//java/jetty:jetty_java11_debug_debian10",
-        "gcr.io/{PROJECT_ID}/python3:latest": "//experimental/python3:python3",
-        "gcr.io/{PROJECT_ID}/python3:nonroot": "//experimental/python3:python3_nonroot",
-        "gcr.io/{PROJECT_ID}/python3:debug": "//experimental/python3:debug",
-        "gcr.io/{PROJECT_ID}/python3:debug-nonroot": "//experimental/python3:debug_nonroot",
-        "gcr.io/{PROJECT_ID}/python2.7:latest": "//experimental/python2.7:python27",
-        "gcr.io/{PROJECT_ID}/python2.7:debug": "//experimental/python2.7:debug",
+        "gcr.io/{PROJECT_ID}/python3:latest": "//experimental/python3:python3_debian9",
```

Comment:
maybe @chanseokoh knows, did we not ever do debian10 images for python? Is it because experimental?

Reply:
AFAIK this file isn't actually used for anything real, since `cloudbuild.yaml` enumerates all the targets explicitly, so I'd take this with a grain of salt.

Reply:
Yeah, `container_bundle` is to build a local tarball with all the images for `docker save` and might be useful for some people during development. (I don't use it.) I tried to sync this with `cloudbuild.yaml`, but I guess I missed Python Debian 10 here.

Reply:
I believe the original intent was to allow `cloudbuild.yaml` to use the bundle in conjunction with a `container_pushall` rule so that the `cloudbuild.yaml` could just becomes `bazel run //:pushall`, however, that would make us beholden to the absence of multi-arch support in rules_docker, so I'm sort of glad we're not stuck with that.

I pointed @jonjohnsonjr at this, so perhaps if/when we adds multi-arch to rules_docker, we can consolidate a bit here and `cloudbuild.yaml` can simplify dramatically 🤩

Reply:
Oh yeah, I remember opening #286.